src/refdb_fs.c

```diff
@@ -1399,16 +1399,14 @@ static int refdb_fs_backend__delete_tail(
 		goto cleanup;
 	}
 
-	if ((flags & GIT_REFDB__SKIP_REFLOG) == 0) {
```

Comment:
It would be nice to have an explanation here why we'd always want to write that log in the commit message. At least I'm a bit clueless why that should be the case, when not writing it has been requested

Reply:
IIRC this is so the HEAD reflog gets its `0124adbc 00000000 sig Branch: renamed foo to bar`, which marks the deletion (the first part of the "renaming" pair). But I'm not quite sure anymore 😞.